atlasdb-config/src/main/java/com/palantir/atlasdb/factory/ServiceDiscoveringAtlasSupplier.java

```diff
@@ -45,26 +46,31 @@
     private static final Logger log = LoggerFactory.getLogger(ServiceDiscoveringAtlasSupplier.class);
     private static final ServiceLoader<AtlasDbFactory> loader = ServiceLoader.load(AtlasDbFactory.class);
 
+    private static final double GOLDEN_RATIO = (Math.sqrt(5) + 1.0) / 2.0;
+    private static final int QUICK_MAX_BACKOFF_SECONDS = 1000;
```

Comment:
Cassandra startup operations include KVS creation, DC sanity checks etc. This is expected to take over 3-4s and depends on factors like network latency, cache warming on C*.. However, if the DB is not up at all it will fail faster. Should we make the cap configurable.. Currently we start at 1s and increase exponentially reaching the cap after around 20 retries.